python-uwsgi/Dockerfile

```diff
@@ -1,18 +1,31 @@
-from unit9/base:latest
-maintainer Kamil Cholewiński <kamil.cholewinski@unit9.com>
+FROM unit9/base:latest
```

Comment:
Please don't change capitalisation, stick to the current convention (whatever the convention is)